src/main/java/com/k3ntako/HTTPServer/controllers/SimplePost.java

```diff
@@ -8,17 +8,32 @@
 
 public class SimplePost implements ControllerInterface {
   private FileIOInterface fileIO;
+  private UUIDInterface uuid;
 
-  public SimplePost(FileIOInterface fileIO) {
+  public SimplePost(FileIOInterface fileIO, UUIDInterface uuid) {
     this.fileIO = fileIO;
+    this.uuid = uuid;
   }
 
-  public Response getResponse(RequestInterface request) throws IOException {
+  public Response getResponse(RequestInterface request) throws IOException, HTTPError {
     var body = request.getBody();
-    Path path = FileSystems.getDefault().getPath("./data/simple_post.txt");
+    validateBody(body);
+
+    var fileUUID = uuid.generate();
+    var filePath = "./data/" + fileUUID + ".txt";
+    Path path = FileSystems.getDefault().getPath(filePath);
```

Comment:
Think about moving this to fileIO or maybe it's own helper class. It would be cool to have something like `fileIO.generateName(...)` or even better, something like `fileIO.saveFile(dataToBeSaved)` that just returns the new file name and if it was a successful save or not. Leaning towards it being a separate class because this is specific to our current implementation for fileIO.